src/Autosuggest.vue

```diff
@@ -143,6 +155,9 @@ export default {
     }
   }),
   computed: {
+    component_attr_id_autosuggest () {
+      return `autosuggest-${this._uid}`;
```

Comment:
We should not rely on _uid as it is internal to vue and not a public api for components to consume, see: https://github.com/vuejs/vue/issues/5886. What we should do instead is default it to `autosuggest` so we can maintain backwards compat without a breaking release. This will also allow you to remove the computed property. 

Also, to be pedantic but hopfeully more clear, we should rename `component_attr_class_autosuggest` to `component_attr_class_autosuggest__container`. 

Can you update the docs as well?